### DISCUSSION TITLE
Battle track score estimations

### DISCUSSION THREAD

**Discussion by dtra** [MKT-Playbook/Hub#9] [General]
Hi

I just downloaded version 1.2.0 (55), noticed the battle score estimates were quite high.
Just providing some numbers to help your calculations, for Cookie Land, I was using Petey Piranha, I managed 169 actions one time, but usually averaged around 110-120 actions, 238 is way too high, boomboxing does not seem to be a thing in Battle Mode.
For Battle Course 1, I'm using Yoshi and my best run was 112 actions, 204 is very unlikely to be possible. Not sure if you're adding the actions to compensate for the well fought bonus, but even then, the score estimates seem way too high.

**Comment by TitusPosiadly** [comment]
Hey there!

Great to see you here.

So, while figuring out the score calculation I decided to go with the action count strat (Trying to get as many actions in the full 3 minutes). While I, unfortunately, didn't have time to thoroughly test this yet, I did a couple of runs on Cookie Land and got 188 actions with a rather mediocre run (1 Bob-omb, 1 mushroom, and 1 green shell frenzy, no NonStop combo, and 20 seconds to spare) so I assumed there is a very high potential with the action count strat. But I am also still in the process of figuring out the best ways to go about this and I am thankful for every input on this. If you (or anyone else stumbling upon this) have any videos of your best runs be sure to link them here. I will do some more tests on Battle Course 1 myself today and will share my new findings here as well.

**Reply by dtra** [reply]
188 actions? Were you using coinbox or lucky 7? Those two skills are much more powerful for battle mode.

**Reply by TitusPosiadly** [reply]
Yes, I was using a lucky seven driver and got 4 single lucky sevens. I did boost those two skills quite a bit but maybe the boomerang has still too many actions - although I thought for the perfect score one would get at least one coinbox from a bot

**Reply by dtra82** [reply]
I think I might’ve seen one coinbox while playing, bot coinboxes don’t seem to be a thing in battle mode, I think only coinbox and lucky 7 are plus skills  in battle mode, maybe mushroom  cannon is above average, not sure.
